libs/custom-paper-handler/src/driver/driver.ts

```diff
@@ -244,11 +233,14 @@ export class PaperHandlerDriver {
   /**
    * Transfers data out on the real time bulk out endpoint.
    */
-  transferOutRealTime(command: Uint8[]): Promise<USBOutTransferResult> {
-    return this.webDevice.transferOut(
-      REAL_TIME_ENDPOINT_OUT,
-      new Uint8Array(command)
-    );
+  transferOutRealTime(requestId: Uint8): Promise<USBOutTransferResult> {
+    // TODO error handling
```

Comment:
Leaving this as a TODO because we'll have a better idea of how to best handle errors when we get more into product development